tests/test_memvid_config.py

```
@@ -0,0 +1,317 @@
+
+        # Retrieval values should be positive
+        assert config["retrieval"]["cache_size"] > 0
+        assert config["retrieval"]["max_workers"] > 0
+
+    def test_get_memvid_config_deep_copy_behavior(self):
+        """Test that the config behaves like a deep copy."""
+        config = get_memvid_config()
+        original_qr_version = config["qr"]["version"]
+
+        # Create a deep copy for comparison
+        config_copy = copy.deepcopy(config)
```

Ruff (0.14.8)

Local variable `config_copy` is assigned to but never used

Remove assignment to unused variable `config_copy`

(F841)